horoscope-app/pages/index.js

```diff
@@ -0,0 +1,133 @@
+import { useState } from "react";
+import axios from "axios";
+
+export default function Home({ data }) {
+  const [res, setRes] = useState(data);
+  const [selectedSign, setSelectedSign] = useState("");
+  const signs = [
```

Comment:
You can define these outside the component as they won't change when the component re-renders.